.github/workflows/tests.yml

```diff
@@ -277,76 +286,74 @@ jobs:
-          chmod +x ${{ github.workspace }}/miniconda.sh
-          ${{ github.workspace }}/miniconda.sh -b -p ${{ github.workspace }}/miniconda
-          source ci/github/activate_conda "${{ github.workspace }}/miniconda/bin/python"
+          conda install -q -y -c defaults --file tests/requirements.txt
+          conda install -q -y -c defaults --file tests/requirements-macos.txt
+          conda install -q -y ${{ env.CONDA_CHANNEL_LABEL }}::conda
+          pip install -e .
 
-      - name: Setup CF MacOS Bits and Select Xcode
+      - name: Setup variables
+        # macOS needs to define $CONDA_BUILD_SYSROOT and $MACOSX_DEPLOYMENT_TARGET to know where to find the SDKs
+        # the Anaconda compilers only support up to macOS 10.15
         run: |
-          set -x -e -u
-          source ci/github/activate_conda "${{ github.workspace }}/miniconda/bin/python"
-          source ci/github/run_conda_forge_build_setup_osx
+          sudo xcode-select --switch /Applications/Xcode_11.7.app
+
+          echo "CONDA_BUILD_SYSROOT=$(xcrun --sdk macosx --show-sdk-path)" >> $GITHUB_ENV
+          echo "MACOSX_DEPLOYMENT_TARGET=$(xcrun --sdk macosx --show-sdk-version)" >> $GITHUB_ENV
 
-      - name: Prepare Test Environment
+      - name: Show info
         run: |
-          set -e -u
-          source ci/github/activate_conda "${{ github.workspace }}/miniconda/bin/python"
-          conda info
-          conda list
-          grep '^#' "${CONDA_PREFIX}/conda-meta/history"
-          conda install python=${{ matrix.python-version }} -y
-          mkdir -p ${{ github.workspace }}/miniconda/locks
-          mkdir -p ${{ github.workspace }}/miniconda/bin
-          chmod -w ${{ github.workspace }}/miniconda/locks
-          conda install -y -c conda-canary/label/dev conda
-          conda config --set always_yes yes
-          conda config --set auto_update_conda False
-          conda info
-          source ci/github/install_conda_build_test_deps
-          pip install -e .
           conda info -a
           conda list --show-channel-urls
 
-      - name: Run Serial Tests
-        if: matrix.test-type == 'serial'
-        run: |
-          set -e -u
-          source ci/github/activate_conda "${{ github.workspace }}/miniconda/bin/python"
-          conda install conda-verify -y
-          pytest --color=yes -v -n 0 --basetemp /tmp/cb_serial --cov conda_build --cov-report xml -m "serial" tests "${PYTEST_REPLAY_OPTIONS[@]+"${PYTEST_REPLAY_OPTIONS[@]}"}"
-
-      - name: Run Parallel Tests
-        if: matrix.test-type == 'parallel'
+      - name: Run tests
         run: |
-          set -e -u
-          source ci/github/activate_conda "${{ github.workspace }}/miniconda/bin/python"
-          conda remove conda-verify -y
-          echo "safety_checks: disabled" >> ~/.condarc
-          echo "local_repodata_ttl: 1800" >> ~/.condarc
```

Comment:
A toplevel
```sh
conda config --set auto_update_conda False
conda config --set local_repodata_ttl: 7200
```
could make sense to avoid unnecessary changes during the test run in case some test runs a `conda install` in the base env and to avoid re-downloading repodata at some point during the CI run (`7200` would be "don't update repodata if it got updated in the last two hours from the former update").

Reply:
I'll move these into `ci/github/.condarc` that we use with our setup-miniconda